eth/v1alpha1/validator.proto

```diff
@@ -179,6 +179,17 @@ service BeaconNodeValidator {
             post: "/eth/v1alpha1/validator/exit"
         };
     }
+
+    // Subscribe to a particular committee ID subnet given validator's duty.
+    //
+    // The beacon node is expected to subscribe to the committee ID subnet given by the request. With this,
+    // beacon node serving attesters can find persistent peers on the subnet to publish attestation,
+    // and beacon node serving aggregator can join the subnet.
+    rpc SubscribeCommitteeSubnet(CommitteeSubnetSubscribeRequest) returns (google.protobuf.Empty) {
+        option (google.api.http) = {
+            post: "/eth/v1alpha1/validator/subscribe"
```

Comment:
this gateway endpoint isnt very descriptive, maybe validator/subnet/subscribe?

Reply:
done